<?xml version="1.0"?>

<launch>

<rosparam param="/uwb_localization_node0/ancsPos"> [-3.0, -3.0, -1.74, 3.0, -3.0, -0.11, 3.0, 3.0, -1.30, -3.0, 3.0, -0.24]</rosparam>
<rosparam param="/uwb_localization_node0/ancsId"> [101, 102, 105, 106]</rosparam>

 <!-- rcmEkfLocalization node -->
  <node respawn="false" pkg="uwb_localization"
        type="uwb_localization_sim_node" name="uwb_localization_node0" output="screen">
            <!-- configuration for the node -->
	<param name="rcmLocalizationRate" type="int" value="50"/>
            <!-- configuration for the p410 -->
	<param name="p410InterfaceChoice" type="int" value="1"/>
	<param name="p410SerialPort" type="string" value="/dev/ttyACM0" />
            <!-- configuration for fcc -->
	<param name="fccSerialPort" type="string" value="/dev/ttyO1" />
	<param name="logEnable" type="bool" value="true" />
            <!-- configuration for the ekf -->
	<param name="enableCalib" type="bool" value="true"/>
	<param name="useLaser" type="bool" value="true"/>
	<param name="medianFilterSize" type="double" value="5.0"/>
  </node>
  
</launch>
